Xamarin.Forms.Platform.iOS/CollectionView/ItemsViewController.cs

```diff
@@ -370,7 +370,7 @@ void UpdateEmptyViewVisibility(bool isEmpty)
 
 					// Now that the native empty view's frame is sized to the UICollectionView, we need to handle
```

Comment:
Should this comment change? In theory, if the native empty view's frame is sized to the UICollectionView, then `_emptyUIView.Frame.ToRectangle()` should work. Looks like `_emptyUIView` still needs to be sized correctly?

Reply:
Mnn, good point. The problem is that when we initialize EmptyView, the Bounds have not yet been adjusted and occupies the entire screen size. I have applied changes and we do the initialization of EmptyView in `ViewWillLayoutSubviews`. When the UICollectionView bounds change, just adjust the size of the EmptyView subview.